hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -322,6 +322,10 @@
   public static final String MAJOR_COMPACTION_JITTER = "hbase.hregion.majorcompaction.jitter";
   public static final float  DEFAULT_MAJOR_COMPACTION_JITTER = 0.50F;
 
+  /** Parameter name for compaction after bulkload */
+  public static final String COMPACTION_AFTER_BULKLOAD_ENABLE =
```

Comment:
Is it possible to add this constants to another place? HConstants is already overloaded...